docs/articles/documentation/reference/testcafe-api/browserconnection/README.md

```diff
@@ -11,11 +11,11 @@ A connection to a [remote browser](../../../guides/concepts/browsers.md#browsers
 
 Use this object to run tests on a remote or a mobile device.
 
-`BrowserConnection` is created with the [testCafe.createBrowserConnection](../testcafe/createbrowserconnection.md) function.
+You can create a new `BrowserConnection` with the [testCafe.createBrowserConnection](../testcafe/createbrowserconnection.md) function.
 
-The [browserConnection.url](url.md) property returns a URL the remote browser should visit in order to connect to the [TestCafe server instance](../testcafe/README.md).
+The [browserConnection.url](url.md) property returns a URL a remote browser can visit to connect to the [TestCafe server instance](../testcafe/README.md).
```

Comment:
> a URL a remote browser can visit to...

Pretty hard to parse grammar here (although it's correct)

*a URL that a remote browser can visit* is easier to understand IMO

Reply:
What about:
> The browserConnection.url property returns a URL that remote browsers can visit to connect to the [TestCafe server instance](../testcafe/README.md).